models/src/main/protobuf/CasperMessage.proto

```diff
@@ -28,23 +28,15 @@ service DeployService {
   rpc listenForContinuationAtName(ContinuationAtNameQuery) returns (ListeningNameContinuationResponse) {}
 }
 
-message PhloLimit {
-  int64 value = 1;
-}
-
-message PhloPrice {
-  int64 value = 1;
-}
```

Comment:
Why did you remove that? I introduced those  to not mix the `limit` and `price` in the code. I understand that we will move away from the _limit_ as we now take all available REVs and transform them with `rev_amount * price` but still - it was safer.

Reply:
Whoops my bad. I didn't think it was a useful distinction.